features/ui/sidebar-navigation/sidebar-navigation.tsx

```diff
@@ -146,6 +146,11 @@ const LinkList = styled(List)`
 
 const CollapseMenuItem = styled(MenuItemButton)`
   display: none;
+  ${(props) =>
+    props.isCollapsed &&
```

Comment:
Interesting, I would have thought that TypeScript would complain here. I'd recommend typing the `CollapseMenuItem` (aka defining the `isCollapsed` prop with TS here). You can find an example of how that's done in this file.

Reply:
sorry for breaking your test haha

Reply:
I had a look at the code now and turns out I was wrong. The `MenuItemButton` component already accepts the `isCollapsed` prop and since `CollapseMenuItem` extends that component it's not necessary to type it.

Sorry for the confusion :)